java/opendcs/src/main/java/decodes/tsdb/algo/RelativeHumidity.java

```diff
@@ -12,10 +12,13 @@
 public class RelativeHumidity extends AW_AlgorithmBase
 {
 
-	@Input public double temperature;
-	@Input public double dewPoint;
+	@Input
+	public double temperature;
+	@Input
+	public double dewPoint;
 
-	@Output public NamedVariable output = new NamedVariable("output", 0);
+	@Output(type = Double.class)
```

Comment:
This is arguably "better", but Double is the default type and thus not actually required.

However; I suggest leaving in as it does make our future intention of supporting additional output types more clear.